autoPyTorch/api/base_task.py

```diff
@@ -1791,6 +1792,8 @@ def score(
                 The test predictions
             y_test (np.ndarray):
                 The test ground truth labels.
+            score_kwargs: Any
+                additional arguments for computing the scores. Some metrics might require special arguments
```

Comment:
could you add a note saying something like: Currently, needed for time series forecasting tasks. See `autoPyTorch/api/time_series_forecasting.py`